src/PrefsAct.scala

```diff
@@ -9,13 +9,17 @@ import _root_.android.preference.PreferenceActivity
 import _root_.android.preference.PreferenceManager
 import _root_.android.view.{Menu, MenuItem}
 import _root_.android.widget.Toast
+import android.util.Log
+
 import java.text.SimpleDateFormat
 import java.io.{File, PrintWriter}
 import java.util.Date
 
 import org.json.JSONObject
 
 class PrefsAct extends PreferenceActivity {
+	val TAG = "APRSdroid.PrefsAct"
```

Comment:
Minor nit-pick, this would normally be a Java `private final static` and I believe it's currently an instance variable. I'd have to look up what the Scala version of that is. I believe I copied this from the style used elsewhere, however.